.travis.yml

```diff
@@ -39,14 +29,22 @@ env:
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*2of2"
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*1of2 project/transitive-plugins"
 
-notifications:
-  email:
-    - eugene.yokota@typesafe.com
-    - joshua.suereth@typesafe.com
+matrix:
+  fast_finish: true
+  include:
+    - env: SBT_CMD="scalafmtDiffTest"
```

Comment:
I don't understand why we use the sbt plugin while there's a bootstrap script that runs really fast in comparison with it. The sbt plugin doesn't buy us anything, it's just an alias to the command line tool.
 
I would propose to put the bootstrap script in the root project directory instead.